fish.go

```diff
@@ -158,6 +160,27 @@ func (a *App) prepareFishFlags(
 	return completions
 }
 
+func fishAddFileFlag(
+	flag Flag,
+	completion *strings.Builder,
+) {
```

Comment:
Is gofmt happy with this?

Reply:
Yes :)

Reply:
Looks fairly non-standard. There aren't that many arguments that it would require this ackward breaking. Is this standard in this project or are you bringing your own twist to this?

Reason I am asking is that format consistency is important in a project with multiple collaborators

Reply:
It is standard in this file, but since I'm the main author I would say it is just my own twist. I will align with the project standard (single line) and change the other functions, too.